scrape.py

```diff
@@ -42,6 +42,20 @@
 
 # All the files we want to list
 files = [
+    # B2GDroid
```

Comment:
I’ve commented on the bug, as I think desktop should be first, but I’ll see what others think.